src/components/MyKiva/LendingStats.vue

```diff
@@ -1,14 +1,92 @@
 <template>
-	<div>
-		<h3
-			class="tw-text-primary"
-		>
+	<div class="tw-mb-6">
+		<h3 class="tw-text-primary tw-mb-2">
 			Ready to grow your impact?
 		</h3>
-		<p
-			class="tw-text-base"
-		>
+		<p class="tw-text-base">
 			Next steps for you based on your lending history
 		</p>
 	</div>
+	<div class="tw-bg-white tw-rounded tw-shadow tw-p-4 tw-w-full" style="max-width: 800px;">
+		<div class="tw-mb-4">
+			<!-- Pill above the list -->
+			<span
+				class="tw-inline-flex tw-items-center tw-justify-center tw-gap-[10px] tw-mb-4"
+				title="Your lending reach"
+				style="
+					padding: 8px 12px;
+					border-radius: 12px;
+					background: #EDF4F1;
+					color: #000;
+					font-family: 'Kiva Post Grot', sans-serif;
+					font-size: 14px;
+					font-weight: 621;
+					line-height: 1;
```

Comment:
We typically only use inline "style" tags for maybe 1 or 2 declarations that we don't have tailwind defs for. Instead of stacking styles, please move to the style block and "@apply" tailwind as much as possible.